Changes.md

```diff
@@ -13,6 +13,11 @@ Fixes
 
 - Resize : Fixed bug which caused unwanted image distortion when changing pixel aspect ratio.
 
+API
+---
+
+- Handle : Adds a convenience constructor for `LinearDrag` that creates a drag in an arbitrary direction on the camera plane.
```

Comment:
Adds -> Added.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/3438/commits/2194a82fc8b193c15f6c21c938087c4909fd6052.